src/legacy/core_plugins/vis_type_table/public/table_vis_type.ts

```diff
@@ -87,14 +88,13 @@ export const createTableVisTypeDefinition = (dependencies) => {
           }),
           min: 0,
           max: 1,
-          aggFilter: ['!filter']
-        }
-      ])
+          aggFilter: ['!filter'],
+        },
+      ]),
     },
     responseHandler,
-    hierarchicalData: function (vis) {
+    hierarchicalData: (vis: any) => {
```

Comment:
We have a `Vis` interface in `ui/vis` which we should use here. (Currently it won't give us added type safety since I believe it's just `any`, but to future-proof I'd prefer we use that type here)

Reply:
np, done